src/calculateRentalCost.js

```diff
@@ -3,8 +3,25 @@
  *
  * @return {number}
  */
+
+const SHORTTERM = 3;
+const LONGTERM = 7;
+const rentperday = 40;
```

Comment:
```suggestion
const SHORT_TERM = 3;
const LONG_TERM = 7;
const RENT_PER_DAY = 40;
```
Constants should be in UPPER_SNAKE_CASE